core/src/main/resources/hudson/PluginManager/sidepanel.groovy

```diff
@@ -28,7 +28,7 @@ l.header()
 l.side_panel {
     l.tasks {
         l.task(icon:"icon-up icon-md", href:rootURL+'/', title:_("Back to Dashboard"))
-        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"), permission:app.MANAGE, it:app)
+        l.task(icon:"icon-gear2 icon-md", href:"${rootURL}/manage", title:_("Manage Jenkins"))
```

Comment:
I can't see any reason for a permission check here, (and manage can't see this view anyway)

Reply:
The only issue is that it will become available to any user, and hence it will pollute UI for non-admin users

Reply:
But non admin users can’t get here, this is under plugin manager?

Reply:
This is a view fragment which won't be accessible thanks to [JEP-220](https://github.com/jenkinsci/jep/blob/master/jep/220/README.adoc).